src/sentry/static/sentry/app/components/events/interfaces/spans/header.tsx

```diff
@@ -415,6 +415,13 @@ class ActualMinimap extends React.PureComponent<{trace: ParsedTraceType}> {
 
     const spanChildren: Array<RawSpanType> = childSpans?.[getSpanID(span)] ?? [];
 
+    // Mark descendents as being rendered. This is to address potential recursion issues due to malformed data.
+    // For example if a span has a span_id that's identical to its parent_span_id.
+    childSpans = {
```

Comment:
Why did you choose to mutate the prop like this instead of redeclaring a new variable?

Reply:
Because it's cool 😎